plasma_framework/test/src/vaults/EthVault.test.js

```diff
@@ -115,6 +117,24 @@ contract('EthVault', ([_, alice]) => {
                 'Must have only one output.',
             );
         });
+
+        it('deposit verifier waits a period of time before takes effect', async () => {
```

Comment:
I hardly like the idea of maintaining two sets of tests for both ERC20 tokens and Eth .... maybe we can extract a high level abstraction so there is only one test set

Reply:
I agree that both tests are very similar the thing is that `swap....` is called in the `deposit` which specific vault implement by itself. To be really sure I haven't forgotten any of them - here we're both  :open_hands: